lib/Service/ContactsIntegration.php

```diff
@@ -106,12 +106,14 @@ public function getMatchingRecipient(string $term): array {
 	 * @return false|null|string
 	 */
 	public function getPhoto(string $email) {
-		$result = $this->contactsManager->search($email, ['EMAIL']);
-		if (count($result) > 0) {
-			if (isset($result[0]['PHOTO'])) {
-				return $this->getPhotoUri($result[0]['PHOTO']);
+		$result = $this->contactsManager->search($email, ['EMAIL'], ['escape_like_param' => false]);
```

Comment:
shouldn't this line be part of https://github.com/nextcloud/mail/pull/6714?

Reply:
I then clarified in the chat about the separation of pr (`getPhoto` In one PR, `doSearch` in another PR).

 Maybe we didn't understand each other? You probably wanted to "have a closer look at" `['escape_like_param' => false]` parameter ?

Reply:
> Maybe we didn't understand each other? You probably wanted to "have a closer look at" `['escape_like_param' => false]` parameter ?

Exactly. I want to be sure the turned off escaping doesn't have unexpected side effects